posthog/models/plugin.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib.postgres.fields import JSONField
```

Comment:
I believe this has been deprecated in favor of [JSONField](https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.JSONField) (imported directly from `django.db.models`)

Reply:
This is only in Django 3.1. We're still on 3.0.7 and this JSONField is imported like so elsewhere as well.

![image](https://user-images.githubusercontent.com/53387/97703911-b7326980-1ab1-11eb-9498-4276014c94cb.png)

I'd rather not upgrade django in this already long PR :D